json-schema-generator/build.gradle

```diff
@@ -0,0 +1,26 @@
+plugins {
+    id("io.micronaut.build.internal.json-schema-module")
+}
+
+dependencies {
+    compileOnly(mn.micronaut.core.processor)
+
+    implementation(mnSourcegen.micronaut.sourcegen.model)
+    implementation(mnSourcegen.micronaut.sourcegen.generator)
+    implementation(mnSourcegen.micronaut.sourcegen.generator.java)
```

Comment:
should this directly depend on the `java` impl?

Reply:
The Source generator for Java is in that package so I need it for tests.